openreview/conference/invitation.py

```diff
@@ -966,6 +966,73 @@ def set_revise_review_invitation(self, conference, reviews, name, start_date, du
         for review in reviews:
             self.client.post_invitation(ReviewRevisionInvitation(conference, name, review, start_date, due_date, additional_fields, remove_fields))
 
+    def set_reviewer_reduced_load_invitation(self, conference, options = {}):
+
+        reduced_load_invitation_reply = {
+            'forum': None,
+            'replyto': None,
+            'readers': {
+                'values-copied': [
+                    conference.get_id()
+                ]
+            },
+            'writers': {
+                'values-copied': [
+                    conference.get_id()
+                ]
+            },
+            'signatures': {
+                'values-regex': '\\(anonymous\\)|~.*'
+            },
+            'content': {
+                'user': {
+                    'description': 'Email address or OpenReview Profile Id',
+                    'order': 1,
+                    'value-regex': '.*',
+                    'required': True
+                },
+                'key': {
+                    'description': 'Email key hash',
+                    'order': 2,
+                    'value-regex': '.{0,100}',
+                    'required': True
+                },
+                'response': {
+                    'required': True,
+                    'description': 'Please confirm your response to the invitation.',
+                    'value-radio': ['Yes', 'No'],
+                    'default': 'Yes',
+                    'order': 3
+                },
+                'reduced_load': {
+                    'description': 'Please select a load that you would be comfortable reviewing.',
```

Comment:
I agree with one minor exception. I am not too keen on calling it **paper**_load since a venue may not have papers. How does **submission_load** sound @zbialecki ?

Reply:
How about we name the field **reviewer_load** and in the description say "Please select the number of submissions that you would be comfortable reviewing"